pkg/cmd/secret/set/http.go

```diff
@@ -75,10 +85,14 @@ func putOrgSecret(client *api.Client, host string, pk *PubKey, opts SetOptions,
 	orgName := opts.OrgName
 	visibility := opts.Visibility
 
-	var repositoryIDs []int
+	var repositoryIDs []string
 	var err error
 	if orgName != "" && visibility == shared.Selected {
-		repositoryIDs, err = mapRepoNameToID(client, host, orgName, opts.RepositoryNames)
+		repos := make([]ghrepo.Interface, 0, len(opts.RepositoryNames))
+		for _, repo := range opts.RepositoryNames {
+			repos = append(repos, ghrepo.New(opts.OrgName, repo))
```

Comment:
Please use `ghrepo.NewWithHost` instead of `New`, which is soft-deprecated

Reply:
Changed as part of https://github.com/cli/cli/pull/4699/commits/e0d9b32d28b0bcbee91cc89085c7586a2db45f99